src/AppBundle/Resources/config/services.yml

```diff
@@ -1,101 +1,87 @@
-parameters:
-    app.installer.class: AppBundle\Installer\AppInstaller
-    app.criteria.menu.class: AppBundle\Criteria\Menu
-    app.controller.menu.class: AppBundle\Controller\MenuController
-    app.criteria.children.class: AppBundle\Criteria\Children
-    app.controller.blog.class: AppBundle\Controller\BlogController
-    app.controller.home.class: AppBundle\Controller\HomeController
-    app.controller.contactform.class: AppBundle\Controller\ContactFormController
-    app.form.type.contact.class: AppBundle\Form\Type\ContactType
-    app.form.contact.form.class: AppBundle\Form\Contact\Form
-    app.entity.message.class: AppBundle\Entity\Message
-    app.mail.sender.class: AppBundle\Mail\Sender
-    app.controller.gallery.class: AppBundle\Controller\GalleryController
```

Comment:
I don't think we had such a clear conclusion.

I'd be leaning (63%) towards not using parameters. Having a parameter for the class name will only allow you to change the class (Super Obvious at the rescue 👊), not arguments, or calls, or anything else. If a service needs to be modified, it can easily be done by redefining it or using a compiler pass that changes the class (or any other change).

The development overhead of using parameters is I think bigger than the uncommon benefit of being able to change the class alone. We could look it up, but it is very probably what the Sf community concluded as well.